src/Chapter06/Listing06.20.ProvidingPropertyValidation.cs

```diff
@@ -34,7 +35,7 @@ public string LastName
                     if(value == "")
                     {
                         throw new ArgumentException(
-                            "LastName cannot be blank.", nameof(value));
+                            "LastName cannot be blank.", "value");
```

Comment:
Why change away from nameof?

Reply:
Updated to match book's text. I'll update everything to be nameof() in a future version